ui/chat/src/ctrl.ts

```diff
@@ -133,8 +133,42 @@ export default class ChatCtrl {
     }
   };
 
+  // Lichess feedback format: 'i18n.key username'
+  translateLichessFeedback = (line: Line): void => {
+    const text = line.t;
+    const parts = text.split(' ');
+    const key = parts[0].split('.');
+    const isAboutMe = `${myUserId()}` == parts[1].toLowerCase();
+    switch (key[1]) {
+      case 'playbanFeedbackAbort':
+        line.t = i18n.site.playbanFeedbackAbort(parts[1]);
+        line.showModal = isAboutMe;
+        break;
+      case 'playbanFeedbackNoStart':
+        line.t = i18n.site.playbanFeedbackNoStart(parts[1]);
+        line.showModal = isAboutMe;
+        break;
+      case 'playbanFeedbackRageQuit':
+        line.t = i18n.site.playbanFeedbackRageQuit(parts[1]);
+        line.showModal = isAboutMe;
+        break;
+      case 'playbanFeedbackSitting':
+        line.t = i18n.site.playbanFeedbackSitting(parts[1]);
+        line.showModal = isAboutMe;
+        break;
+      case 'playbanFeedbackQuickResign':
+        line.showModal = isAboutMe;
+        line.t = i18n.site.playbanFeedbackQuickResign(parts[1]);
+        break;
+    }
+  };
+
   private onMessage = (line: Line): void => {
+    if (line.u === 'lichess' && line.t.startsWith('i18n.')) {
+      this.translateLichessFeedback(line);
+    }
```

Comment:
if we only convert i18n messages when received from the WS, they'll remain untranslated when (re)loading the page.

Reply:
I think I saw the Lichess messages disappear from the chat when reloading ... I have to check

Reply:
Yes this particular message is not stored, but other potentially translatable messages are.